paper-radio-button.html

```diff
@@ -114,6 +114,7 @@
     },
 
     ready: function() {
+      this.toggles = true;
```

Comment:
The property will be surfaced in the doc when the behaviors get surfaced, though. I don't think it's a big problem that the user can configure this to a non-default value.